roles/nighthawk/tasks/passthrough-tasks.yml

```diff
@@ -0,0 +1,24 @@
+---
+# tasks for passthrough termination type.
+
+- name: Create service for passthrough type server pods
+  k8s:
+    definition: "{{ lookup('template', 'https-service.yml.j2') | from_yaml }}"
+  with_sequence: start=0 count={{ workload_args.number_of_routes | default(1)  | int }}
+  vars:
+    type: passthrough
+  when: 
+    - workload_args.serviceip is defined and workload_args.serviceip
+    - ( workload_args.service_type | default("ClusterIP") == "NodePort" )
```

Comment:
Remove this line?

Reply:
Done as we have decided to keep clusterIP as default